docs/glossary_concepts.txt

```diff
@@ -0,0 +1,177 @@
+* **late-tabulation ballot** - A ballot which is tabulated after the CVR
+* **list of ballots to audit** TBD
+* **load testing** TBD
+* **mail ballot** TBD
+* **mandatory contest** - A contest which is subject to a risk limit and is
+* **manual ballot adjudication** TBD
+* **margin** TBD
+* **non-voter-verifiable ballot** (NVVB) - A ballot for which there is
+* **offeror** - A vendor that submits a responsible bid for this
+* **operational start date** - When the State authorizes Contractor to
+* **opportunistic audit** - An auditing technique designed to efficiently
+* **opportunistic contest** - A contest to be audited opportunistically.
+* **original ballot** TBD
+* **other personnel** - Individuals and Subcontractors, in addition to Key
+* **overstatement of the margin** TBD
+* **overvote** TBD
+* **paper ballot** TBD
+* **password** TBD
+* **penetration testing** TBD
+* **phantom ballot** A ballot which is listed in the manifest, but is
+* **political party** TBD
+* **post-election (historical, random) audit** TBD
+* **press** TBD
+* **pseudo-random number generator** - A random number generator
+* **public audit meeting** The meeting where the random seed (and
+* **random seed** - A random seed (or seed state, or just seed) is data, such
+* **random** TBD
+* **recount** TBD, a procedure under Colorado law that happens *after*
+* **regional training** TBD
+* **reported outcome** - The set of contest winners published by the ENR
+* **repository** TBD
+* **requirement** TBD
+* **resilient canvass framework** - A fault-tolerant approach to conducting
+* **responsible bid** - A bid from a vendor that can responsibly
+* **risk limit** - The pre-specified minimum chance of requiring a full
+* **risk-limiting audit (RLA)** - A procedure for manually checking a
+* **role** TBD
+* **sample size** TBD including **initial sample size**
+* **scanner ID number** TBD
+* **scanner** has an ID number
+* **settled contest** TBD involving having achieved risk limit.  Note need
+* **start-up period** - The period starting on the Effective Date and
+* **state administrator** - The designated representative(s) of the
+* **state-wide dashboard** TBD
+* **storage bin** (aka transfer case, ballot box, secured container)
+* **stray mark** TBD
+* **system architecture** TBD
+* **tabulated ballots** - Paper ballots that have been scanned on a
+* **tabulation** - Interpretation of expressions of voter intent and aggregation
+* **tool back-end** TBD
+* **tool front-end** TBD
+* **two-factor authentication** - Defined as two out of the three
+* **uncontested contest** TBD
+* **understatement of the margin** TBD
+* **undervote** TBD
+* **uploading interface** A part of the county dashboard.
+* **user experience (UX)** TBD
+* **user interface (UI)** TBD
+* **username** TBD
+* **voter marking** TBD
+* **voting method** TBD
+* **voting system** TBD
+* **working group** TBD
+* **wrong outcome** - When the reported outcome for a given contest does not match the
```

Comment:
Thanks for those reflections.  We'll go ahead and add them to the glossary and specification, realizing that some may end up being synonyms and others may end up being eliminated, just like all other concepts introduced and evaluated.